### PR TITLE
initial  copy of my past project

### PR DESCRIPTION
**Pull request by osmanwako**
# Required Change
# Keep it up 
- good Github follow 
- some of the good use of semantic structure 
# please change all of 
- please fix linter errors
-  folder arrangement is not good
-  remove the more unnecessary commented line

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,229 +58,115 @@ After you're finished please remove all the comments and instructions!
...
+If you are interesting to this port folio please join microverse
+Read w3chool website html and CSS tutorials
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
```
Comment (osmanwako) [inline README.md:258]: I kindly request you to list  who would you like to give Acknowledgement concerning this project achievement here instead empty comment

File: README.md [hunk 2]
```diff
@@ -60,229 +58,115 @@ After you're finished please remove all the comments and instructions!
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
+## 📝 License <a name="MIT.md"></a>
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (osmanwako) [inline README.md:286]: I think this line of code has no value within your code can you remove it

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  align-items: center;
+  text-align: center;
+  margin-left: 20px;
+}
+
+nav {
+  font-weight: 400;
+}
+
+nav > ul {
```
Comment (osmanwako) [inline css/style.css:45]: I think this structure of style selector is not professional way and also makes Linter errors to your CSS file example: 

1. Expected selector "ul" to come before selector "nav > ul"   no-descending-specificity

File: index.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#Features">Features</a></li>
+            <li><a class="nav-link" href="#How It Works">How It Works</a></li>
+            <li><a class="nav-link" href="">Pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div style="margin-top: 100px; margin-left: 50px">
```
Comment (osmanwako) [inline index.html:44]: The professional way of coding CSS style is not recommended for the inline CSS .  would you add it to the external CSS file?
``` suggestion
      <div style="margin-top: 100px; margin-left: 50px">
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/y8Yv4pnO7qc"
+          title="Roman Carnival Overture Op. 9 for Five Trombones"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+      </div>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
```
Comment (osmanwako) [inline index.html:55]: If you forget the method name of the form  I would like to remind you.
```suggestion
      <form id="form" action="https://www.freecodecamp.com/email-submit">
```

File: index.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+        <input
```
Comment (osmanwako) [inline index.html:56]: Validations are supporter and essential for client-side users and server-side owners to interchange valid data please add all expected  validation for all input element
```suggestion
        <input
```

File: README.md [hunk 7]
```diff
@@ -60,229 +58,115 @@ After you're finished please remove all the comments and instructions!
...
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (osmanwako) [inline README.md:212]: - Would  you remove this broken link from your readme or replace the broken link with your Twitter account  else  you may update your proper  link Twitter URL address

**Review by osmanwako — COMMENTED** [review]
# Required to change
- linter errors
-  folder arrangement is not good
-  more unnecessary commented line
# Keep it up 
- good Github follow 
- some of the good use of semantic structure

File: README.md (hunk 7 above)
Comment (anagudelogu) [reply README.md:212]: - [ ] You did well working on this milestone! 💯 However, let's try to be clear when we are requesting a change, a good structure would be to point out the error politely first and then explain briefly why the reviewed person would benefit from changing it!

So in this case, for example, you could rephrase this to be something like:

I went through your social links and noticed your Twitter link is broken **(point out the problem)**, please consider fixing it or removing it as having broken social media links in your project could make future recruiters have a harder time connecting with you. **(give a brief explanation on why this change could benefit the reviewed)** 

Please consider editing this comment and applying this suggestion! 🤓 🔧

File: README.md (hunk 1 above)
Comment (anagudelogu) [reply README.md:258]: - [ ] This comment lacks an explanation, please provide a reason "why" the reviewed person should apply this change. 🤓 🔧

File: css/style.css (hunk 3 above)
Comment (anagudelogu) [reply css/style.css:45]: - [ ] Please be clear on what you're requesting, and also provide an explanation here too. You're pointing out the error but you're not providing any kind of information to the student about why this should be fixed. Please consider editing this comment and applying the suggestions! 🤓 🔧

File: index.html (hunk 4 above)
Comment (anagudelogu) [reply index.html:44]: - [ ] Saying "The professional way" is too vague of a reason to request a change, please be specific on why using inline styles is not recommended and why the reviewed person could benefit from moving it into an external CSS file! 🤓 🔧

File: index.html (hunk 5 above)
Comment (anagudelogu) [reply index.html:55]: - [ ] Please be clear in your comments, you pointed out that the person being reviewed forgot the method attribute of the form, but you're not requesting any change with that information! 🤓 🔧 

>This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?
>
>This is a vague comment: The README is not descriptive, why don't you make it better?

File: index.html (hunk 6 above)
Comment (anagudelogu) [reply index.html:56]: This one is better, you're asking to add validations (clear) and stating that they are essential for the client and server. Good job! 🤓 🚀

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @osmanwako, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub's review features! 🥇 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 7 above)
Comment (osmanwako) [reply README.md:212]: You did well working on Github and Linked links, however, the Twitter link is broken link. Anyone who wants to contact you through Twitter could not contact you. Please update the broken Twitter link with your Twitter URL address.

File: README.md (hunk 1 above)
Comment (osmanwako) [reply README.md:258]: If there are contributors, mentors,  advisors, or others who help you with this project, you can explain their acknowledgment here instead of empty comments.
- Polite: To show your gratitude to people who have given up their time to review your paper or offer comments. It is the polite thing to do
- Ethics: some of your arguments or data might have originated with someone else and you are acknowledging that rather than passing it off as your own
- It can show the editor that your work has already been subject to an informal review

File: css/style.css (hunk 3 above)
Comment (osmanwako) [reply css/style.css:45]: You are using a selector specificity ("nav > ul" ) here. but I think  it is not proper descending specificity 
and also  Selector Specificity is not recommended due to 
-The specificity algorithm calculates the weight of a CSS selector to determine which rule from competing CSS declarations gets applied to an element. if the Selector Specificity has no appropriate descending Specificity the browser going to match more selectors. can you use an attribute specificity instead?

File: index.html (hunk 5 above)
Comment (osmanwako) [reply index.html:55]: For your form elements, You have specified action attributes explicitly that are good things. But the Method attribute has not been specified explicitly. the browser uses the default attribute value means "GET".

The method attribute specifies how to send form data (the form data is sent to the page specified in the action attribute).
The form data can be sent as URL variables (with method="get") or as HTTP post transaction (with method="post")

 # Notes on GET:
Appends form data into the URL in name/value pairs
- The length of a URL is limited (about 3000 characters)
- Never use GET to send sensitive data! (will be visible in the URL)
- Useful for form submissions where a user wants to bookmark the result
- GET is better for non-secure data, like query strings in Google
If you do not agree with above mention please can you use the POST method attribute?
 https://www.w3schools.com/tags/att_form_method.asp read more about post

**Review by osmanwako — COMMENTED** [review]
# Updated Review
- I updated the reviewer's comment on the replay
-  I added more explanation as long as the reviewer's comment

**Review by wayungi — APPROVED** [review]
Hi @osmanwako , 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] No merge conflicts ✔️ 
- [ ]  Good effort ✔️ 
## Optional suggestions
 - [ ] Please you should always fix linter errors in your project before submission
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by osmanwako** [comment]
My first review on my trial project
